Allow changing the link preview settings via the UI when the AI link preview feature is disabled
---
diff --git a/browser/components/preferences/config/tabs-browsing.mjs b/browser/components/preferences/config/tabs-browsing.mjs
index 4f4aa1a0d9..c2fe4aba1c 100644
--- a/browser/components/preferences/config/tabs-browsing.mjs
+++ b/browser/components/preferences/config/tabs-browsing.mjs
@@ -445,22 +445,20 @@ Preferences.addSetting({
 Preferences.addSetting({
   id: "linkPreviewEnabled",
   pref: "browser.ml.linkPreview.enabled",
+});
+Preferences.addSetting({
+  id: "linkPreviewKeyPoints",
+  pref: "browser.ml.linkPreview.optin",
   deps: ["aiControlDefault", "aiControlLinkPreviews"],
   visible: ({ aiControlDefault, aiControlLinkPreviews }) => {
     return (
       window.canShowAiFeature(aiControlLinkPreviews, aiControlDefault) &&
+      // LinkPreview.canShowKeyPoints depends on the global genai pref.
       // @ts-ignore bug 1996860
-      lazy.LinkPreview.canShowPreferences
+      lazy.LinkPreview.canShowKeyPoints
     );
   },
 });
-Preferences.addSetting({
-  id: "linkPreviewKeyPoints",
-  pref: "browser.ml.linkPreview.optin",
-  // LinkPreview.canShowKeyPoints depends on the global genai pref.
-  // @ts-ignore bug 1996860
-  visible: () => lazy.LinkPreview.canShowKeyPoints,
-});
 Preferences.addSetting({
   id: "linkPreviewLongPress",
   pref: "browser.ml.linkPreview.longPress",
